sys/include/net/ng_fib.h

```diff
@@ -70,14 +70,15 @@ void fib_deinit(void);
 /**
  * @brief Registration of a routing protocol handler function
  *
- * @param[in] prefix the prefix handled by the according RP
- * @param[in] prefix_size the prefix size
+ * @param[in] prefix                the prefix handled by the according RP
+ * @param[in] prefix_addr_type_size the bytes used for the prefix address type
```

Comment:
*puts nitpick hat on* Maybe rather “The size of the address type used for the prefix”?